jme3-desktop/src/main/java/com/jme3/system/AWTUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.jme3.system;
+
+import com.jme3.texture.FrameBuffer;
+import com.jme3.texture.Image;
+
+public class AWTUtils {
+    /**
+     * This should be a Temporary solution. FrameBuffer functions and Image-Formats are deprecated.
+     * 
+     * @param width
+     * @param height
+     * @param samples
+     * @return
+     */
+    @SuppressWarnings("deprecation")
+    public static FrameBuffer getFrameBuffer(int width, int height, int samples) {
```

Comment:
Should this have @SuppressWarnings("deprecation")? The warnings are valid. More generally. Why extract this out into a seperate method (if we want to discourage using these deprecated things we don't want to make reuse easier)